addon/components/table-of-contents-plugin/ember-nodes/table-of-contents.ts

```diff
@@ -89,7 +89,7 @@ export default class TableOfContentsComponent extends Component<EmberNodeArgs> {
         );
         const config = this.config[0];
         if (config.scrollContainer) {
-          const scrollContainer: HTMLElement = config.scrollContainer;
+          const scrollContainer: HTMLElement = config.scrollContainer();
```

Comment:
I can't reproduce that error here fyi
"no unsafe assignment of any" errors are often false positives in the webstorm eslint service that are fixed after a restart of that service

In any case it would be a linting error and not a type error

Reply:
Yes disregard this comment. lint error is gone (so false positive) and it didn't make sense anyway because `config` has correct typing.